### PR TITLE
Contact Form w/ Mailgun

### PR DESCRIPTION
**Pull request by thaitwo**
Hey, whenever you have some free time, can you please take a look over my code? Thank you! 🐤 🔑 🍶

### CODE REVIEW

File: resources/scss/_form.scss [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.form {
+  width: 420px;
+  display: block;
+  float: left;
+
+  & input[type=email],
```
Comment (alexwolfe) [inline resources/scss/_form.scss]: What do you have `&` symbols here? You shouldn't have any, something ain't right 😄

File: routes/index.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
 var express = require('express');
 var router = express.Router();
+var api_key = 'key-d0a20155f81d1b9337d53b4f282f306a';
```
Comment (alexwolfe) [inline routes/index.js]: we're gonna want to take this out of your public repo

File: routes/index.js [hunk 3]
```diff
@@ -13,7 +16,65 @@ router.get('/about', function(req, res, next) {
...
+      title: 'Contact',
+      errors: errors,
+      name: name,
+      email: email,
+      message: message
+    })
+  }
+  else {
+    // SEND TO EMAIL
+    mailgun.messages().send(data, function (error, body) {
```
Comment (alexwolfe) [inline routes/index.js]: I think the res.render should be inside the function. Basically you call it once maigun is done sending: 

```
mailgun.messages().send(data, function (error, body) {
  if(error) {
    // SHOW CONTACT PAGE WITH ERROR MESSAGE
  }
  else {
    // SUCCESS
    res.render('contact-success', {
      title: 'Contact',
      name: name,
      email: email,
      message: message
    });
  }
});
```

**Comment by thaitwo** [comment]
@alexwolfe I just committed the updated coding for the contact form. Can you take a quick glance and merge to master if you think it's ready, when you have some time? Thanks! 🍜

File: README.md [hunk 4]
```diff
@@ -1,2 +1,9 @@
-# tu-photography
+# photography portfolio
```
Comment (alexwolfe) [inline README.md]: 👍

File: resources/scss/_form.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    border-radius: 2px;
+  }
+
+  ul li {
+    margin-bottom: 10px;
+  }
+}
+
+
+#form-alert {
```
Comment (alexwolfe) [inline resources/scss/_form.scss]: classes please 💃

File: resources/scss/_form.scss [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  font-family: $font-family;
+  font-size: $font-size-1x;
+  color: $red-800;
+  font-weight: 400;
+  box-sizing: border-box;
+  line-height: 18px;
+  margin-top: 10px;
+}
+
+#form-success {
```
Comment (alexwolfe) [inline resources/scss/_form.scss]: same, use a class 😛

File: views/contact-success.ejs [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  	<h1>THANK YOU!</h1><br>
+  	<p>Your message has been delivered successfully.</p><br>
+
+    <div class="cta">
+      <a class="button button-secondary" href="/gallery">VIEW GALLERIES</a>
+    </div>
+  </div>
+  <img class="col-6 is-full-width is-rounded" src="/images/marlette.jpg">
+
+   <div class="clear"></div>
```
Comment (alexwolfe) [inline views/contact-success.ejs]: indent off one space 👾

**Comment by thaitwo** [comment]
@alexwolfe It should be all good to go. I made the form responsive as well. Please merge if you think it's good enough. THANK YOU!! 😄  🍯

File: README.md [hunk 8]
```diff
@@ -1,9 +1,32 @@
 # photography portfolio
-Personal photography website
+This project is my personal photography portfolio featuring multiple galleries, a gallery viewer, and a contact form. The website is also responsive and mobile-friendly.
 
-[![thaitwo](/images/thaitwo-homepage)](http://www.thaitwo.com)
+[![thaitwo](/images/thaitwo-homepage.png)](http://www.thaitwo.com)
```
Comment (alexwolfe) [inline README.md]: This won't work, you'll need the full url: `http://www.thaitwo.com/images/thaitwo-homepage.png`

**Comment by thaitwo** [comment]
@alexwolfe alrighty, url has been updated. Thanks for the catch! Please merge. Thanks!!

**Comment by alexwolfe** [comment]
@thaitwo Ship it son! 😎 🚢

**Pull request merged**